docs/components/BarChartDocs.js

```diff
@@ -114,7 +114,7 @@ class BarChartDocs extends React.Component {
         <p>Object containing the desired padding on the SVG to account for axis labels and tooltips.</p>
         <p>Default:</p>
         <Markdown lang='js'>
-  {`
+          {`
```

Comment:
hmm good question, i'm wondering if the eslint autofix is not working as we need it to. looking